aiohttp/connector.py

```diff
@@ -576,6 +576,7 @@ class TCPConnector(BaseConnector):
         after each request (and between redirects).
     limit - The total number of simultaneous connections.
     limit_per_host - Number of simultaneous connections to one host.
+    enable_cleanup_closed - Disable clean-up closed ssl transports.
```

Comment:
This sounds like a bit confusing. Flag named as "enable", but it does "disable". How about:
`enable_cleanup_closed - Enables clean-up closed ssl transports. Disabled by default.`